docs/basic-usage/ssl.md

```diff
@@ -124,13 +124,13 @@ You also need to disable SSL verification.
     'options' => [
         'cluster' => env('PUSHER_APP_CLUSTER'),
         'encrypted' => true,
-        'host' => '127.0.0.1',
-        'port' => 6001,
-        'scheme' => 'https',
+        'host' => env('PUSHER_APP_HOST', '127.0.0.1'),
+        'port' => env('PUSHER_APP_PORT', 6001),
+        'scheme' => env('PUSHER_APP_SCHEME', 'http'),
```

Comment:
Should add the path option in here too. It is how we handle things locally using the same  local host for sockets
`
     'path' => env('PUSHER_PATH'),
`

Reply:
According to this: https://github.com/pusher/pusher-http-php/blob/master/src/Pusher.php#L56 I don't see that `path` is a valid parameter. 🤔

Reply:
The app itself should have the path, as in this review: https://github.com/beyondcode/laravel-websockets/pull/447#pullrequestreview-477174606